### DISCUSSION TITLE
How do you convert RGB to OKLCH?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [Evercoder/culori#252] [Q&A]
Testing some symmetric conversion to make sure things are what I expect

I'm just testing things out on the docs site here: https://culorijs.org/api/#formatHex
in the browser console.

![image](https://github.com/user-attachments/assets/333aba3a-ea9e-468d-8c00-ab81655389a1)

```js
culori.formatHex(culori.parseHex('#0074ff'))
// "#0074ff" 

culori.formatHex(culori.convertRgbToLab(culori.parseHex('#0074ff')))
// "#0074ff"  

culori.formatHex(
  culori.convertLabToLch(
    culori.convertRgbToLab(
      culori.parseHex('#0074ff')
    ), 
    'oklch'
  )
)
// "#ff00ff" -- wrong
```
in that last one, if I omit the `oklch` argument, which converts to regular `lch`, formatHex returns the original hex, as expected

**Comment by NullVoxPopuli** [comment]
ah-ha!

```js
culori.formatHex(
  culori.convertLabToLch(
    culori.convertRgbToOklab(
      culori.parseHex('#0074ff')
    ), 
    'oklch'
  )
) 
```

This does it

**Reply by danburzo** [reply]
Yes, that’s what I was meaning to point out. In your original example, CIELAB (`lab`) and Oklab (`oklab`) were being mixed.
